compile.sh

```diff
@@ -1,28 +1,28 @@
 #!/usr/bin/env bash
-[ -z "$PHP_VERSION" ] && PHP_VERSION="8.2.17"
+[ -z "$PHP_VERSION" ] && PHP_VERSION="8.2.25"
 
 #### NOTE: Tags with "v" prefixes behave weirdly in the GitHub API. They'll be stripped in some places but not others.
 #### Use commit hashes to avoid this.
 
 ZLIB_VERSION="1.3.1"
 GMP_VERSION="6.3.0"
-CURL_VERSION="curl-8_7_1"
+CURL_VERSION="curl-8_9_1"
```

Comment:
Might be a good idea to open an issue for newer curl so it doesn't get forgotten about